.github/workflows/docker-build.yaml

```diff
@@ -56,28 +56,25 @@ jobs:
           fi
       - name: Log in to the Container registry
         run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ${{ env.REGISTRY }} --username ${{ github.actor }} --password-stdin
-      - uses: webfactory/ssh-agent@v0.7.0
-        with:
-          ssh-private-key: ${{ secrets.STJUDE_SEQERR_READONLY_DEPLOY_KEY }}
       - uses: docker/setup-qemu-action@v2
       - uses: docker/setup-buildx-action@v2
       - uses: docker/build-push-action@v4
         id: build
         with:
           context: ${{ matrix.image }}
-          build-contexts: scripts=./scripts
+          build-contexts: scripts=./docker_scripts
```

Comment:
I actually don't like this change. Part of the reason that we removed embedded scripts was to make it possible for users (such as Charlie) to run them separately. So they're decoupled from the WDL and the Docker images, even though they get included in the docker images.

Reply:
Ok, that sounds fine to me. I'll revert these changes 👍